### ISSUE TITLE
accessibility: adapt-contrib-narrative general fixes

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt_framework#2193]
### Subject of the issue
* Fix disjointed breaking behaviour
* Fix strapline text overflow for ios
* Remove accessibility mode
* Shorten component description
* Remove role="button"
* Remove a11y_text
* Hide image in place of label representation
* Pull through heading values

### Actual behaviour
* Narrative desktop/mobile switching is distributed over multiple functions and less styling, this should be dependent upon one central switching block.
* On iOS inside iframe, the nowrap styling in the narrative strapline div cause the browser to zoom.

**Comment by oliverfoster** [comment]
done
